hydra-node/test/Hydra/NodeSpec.hs

```diff
@@ -149,7 +149,11 @@ createHydraNode signingKey otherParties events = do
   pure $
     HydraNode
       { eq
-      , hn = Network{broadcast = const $ pure ()}
+      , hn =
+          Network
+            { broadcast = const . pure $ ()
+            , modifyPeers = \f -> pure . fst . f $ mempty
```

Comment:
If unused, it would be better to error. Otherwise, this may get tricky to debug if we ever write a test that depends on it without remembering that the implementation is bonkers.